lib/rubocop/cop/rspec/no_expectation_example.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Checks if an example contains any expectation.
+      #
+      # All RSpec's example and expectation methods are covered by default.
+      # If you are using your own custom methods,
+      # add the following configuration:
+      #
+      #   RSpec:
+      #     Language:
+      #       Examples:
+      #         Regular:
+      #           - custom_it
+      #       Expectations:
+      #         - custom_expect
+      #
+      # @example
+      #
+      #   # bad
+      #   it do
+      #     a?
+      #   end
+      #
+      #   # good
+      #   it do
+      #     expect(a?).to be(true)
+      #   end
+      class NoExpectationExample < Base
+        MSG = 'No expectation found in this example.'
+
+        # @!method including_any_expectation?(node)
+        # @param [RuboCop::AST::Node] node
+        # @return [Boolean]
+        def_node_search(
+          :including_any_expectation?,
+          send_pattern('#Expectations.all')
+        )
+
+        # @param [RuboCop::AST::BlockNode] node
+        def on_block(node)
+          return unless example_method_call?(node)
```

Comment:
I think we have an [`example?` helper method](https://github.com/rubocop/rubocop-rspec/blame/master/lib/rubocop/rspec/language.rb#L41). Would it work here?